Source/RN4UE4/RN4UE4GameInstance.h

```diff
@@ -0,0 +1,27 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Engine/GameInstance.h"
+#include "RN4UE4GameInstance.generated.h"
+
+/**
+ * 
+ */
+UCLASS()
+class RN4UE4_API URN4UE4GameInstance : public UGameInstance
+{
+	GENERATED_BODY()
+	
+public:
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Levels")
+		int indexLevel;
```

Comment:
This can be removed

Reply:
I use that variable to check what level have been loaded to persistent level